lib/cartopy/io/shapereader.py

```diff
@@ -299,10 +301,11 @@ def natural_earth(resolution='110m', category='physical', name='coastline'):
     # get hold of the Downloader (typically a NEShpDownloader instance)
     # which we can then simply call its path method to get the appropriate
     # shapefile (it will download if necessary)
+    _version_string = "" if version is None else f"{version}/"
```

Comment:
This doesn't need to be private if it is just used within the method.
```suggestion
    version_string = "" if version is None else f"{version}/"
```